### PR TITLE
Merchant changes.

### PR DESCRIPTION
**Pull request by Kaksisilma**
Same as https://github.com/Alanii/NewHestia/pull/372.

This PR adds a few tweaks to the gunshop merchant:
They now sell a wider selection of weapons, including assault rifles, submachine guns, replica pistols, double-barrel shotguns, and, for a small fortune, the L6 Saw.
They no longer sell the QX-2 shotgun. (This was an oversight when it was first implemented, as is for nearly all alien weapons)
Minor alterations to gun costs.
:cl: Vhbraz
tweak: The guns sold by the gunshop merchant have been tweaked to include a wider array of weapons.
/:cl:

Changelog coming in a second.

### CODE REVIEW

File: baystation12.dme [hunk 1]
```diff
@@ -3228,9 +3232,6 @@
 #include "modular_boh\code\modules\telecomms\encryptionkey.dm"
 #include "modular_boh\code\modules\telecomms\headset.dm"
 #include "modular_boh\code\modules\telecomms\preset.dm"
-#include "modular_boh\code\modules\culture_descriptor\culture\cultures_human.dm"
```
Comment (BurpleBineapple) [inline baystation12.dme:3231]: What's going on over here with the uh.. includes being deleted?

Comment (BurpleBineapple) [reply baystation12.dme:3231]: You might be using an older branch

Comment (Kaksisilma) [reply baystation12.dme:3231]: Should probably be fixed.

**Review by BurpleBineapple — APPROVED** [review]
Approved

**Pull request merged**